varlink/client.py

```diff
@@ -327,6 +329,13 @@ class Client:
 
     handler = SimpleClientInterfaceHandler
 
+    @classmethod
+    @functools.lru_cache(maxsize=1)
+    def _get_base_interface(cls):
+        return Interface(
+            importlib.resources.read_text("varlink", "org.varlink.service.varlink")
+        )
```

Comment:
Should have also started CI after the last push.
```suggestion
        return Interface(importlib.resources.read_text("varlink", "org.varlink.service.varlink"))
```

Reply:
Yup missed that since the ruff-check and ruff-format are not listed by tox -l :/